src/table/utils/__tests__/handle-key-press.spec.js

```diff
@@ -142,6 +263,150 @@ describe('handle-key-press', () => {
     });
   });
 
+  describe('headHandleKeyPress', () => {
```

Comment:
Are these lines only moved, no changes?

Reply:
some changes, as headHandleKeyPress = (
  evt,
  rootElement,
  cellCoord,
  column,
  changeSortOrder,
  layout,
  isAnalysisMode,
  setFocusedCellCoord
) is updated to be headHandleKeyPress = ({
  evt,
  rootElement,
  cellCoord,
  column,
  changeSortOrder,
  layout,
  isAnalysisMode,
  setFocusedCellCoord
})

Reply:
ah ok great